server/src/main/java/com/lunchtime/config/ResourcesPath.java

```diff
@@ -0,0 +1,13 @@
+package com.lunchtime.config;
+
+import org.springframework.context.annotation.Configuration;
+
+import java.io.File;
+
+@Configuration
+public class ResourcesPath {
+
+    public static String getResourcePath() {
+        return new File(System.getProperty("user.dir")).getParent() + "/static/";
```

Comment:
You shouldn't use `getParent()` method, because application can be run in root folder (`/`)
![image](https://user-images.githubusercontent.com/6152066/80370911-b0743100-8899-11ea-887c-b16fcfd0b652.png)
![image](https://user-images.githubusercontent.com/6152066/80370929-bb2ec600-8899-11ea-8286-aaa8a09239be.png)
Or make your code more flexible

In our staging server application runs in folder `/app` by `appuser` and doesn't have rights to create `/static` folder. You also should update docker-compose file - add new `static` volume, which must be mapped to real filesystem
`
volumes:
      - ./server/static:/app/static
`
Without it, all users files (avatars) will be removed after next update